config/routes.rb

```diff
@@ -14,6 +14,14 @@
         get ':id/managed', on: :collection, action: :managed_users
       end
       resources :assignments
+      resources :courses do
```

Comment:
Please maintain consistency by following the syntax used in the existing routes.

Reply:
Okay, I will change the syntax.